src/site/content/en/blog/webtransport/index.md

```diff
@@ -68,56 +53,56 @@ As part of the origin trial process, we're interested in [hearing more](#feedbac
+Generally, running a HTTP/3-compatible server requires less setup and configuration than maintaining a WebRTC server, which involves understanding multiple protocols ([ICE](https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Connectivity#ICE_candidates), [DTLS](https://webrtc-security.github.io/#4.3.1.), and [SCTP](https://developer.mozilla.org/en-US/docs/Web/API/RTCSctpTransport)) in order to get a working transport. WebRTC entails many more moving pieces that could lead to failed client/server negotiations.
 
-The QuicTransport API was designed with the web developer use cases in mind, and should feel more like writing modern web platform code than using WebRTC's data channel interfaces. [Unlike WebRTC](https://bugs.chromium.org/p/chromium/issues/detail?id=302019), QuicTransport is supported inside of [Web Workers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers), which allows you to perform client-server communications independent of a given HTML page. Because QuicTransport exposes a [Streams](https://streams.spec.whatwg.org/)-compliant interface, it supports optimizations around [backpressure](https://streams.spec.whatwg.org/#backpressure).
+The WebTransport API was designed with the web developer use cases in mind, and should feel more like writing modern web platform code than using WebRTC's data channel interfaces. [Unlike WebRTC](https://bugs.chromium.org/p/chromium/issues/detail?id=302019), WebTransport is supported inside of [Web Workers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers), which allows you to perform client-server communications independent of a given HTML page. Because WebTransport exposes a [Streams](https://streams.spec.whatwg.org/)-compliant interface, it supports optimizations around [backpressure](https://streams.spec.whatwg.org/#backpressure).
 
-However, if you already have a working WebRTC client/server setup that you're happy with, switching to QuicTransport may not offer many advantages.
+However, if you already have a working WebRTC client/server setup that you're happy with, switching to WebTransport may not offer many advantages.
 
 ## Try it out
 
-The best way to experiment with QuicTransport is to use [this Python code](https://github.com/GoogleChrome/samples/blob/gh-pages/quictransport/quic_transport_server.py) to start up a compatible QUIC server locally. You can then use this page with a [basic JavaScript client](https://googlechrome.github.io/samples/quictransport/client.html) to try out client/server communications.
+The best way to experiment with WebTransport is to use [this Python code](https://github.com/GoogleChrome/samples/blob/gh-pages/webtransport/web_transport_server.py) to start up a compatible HTTP/3 server locally. You can then use this page with a [basic JavaScript client](https://googlechrome.github.io/samples/webtransport/client.html) to try out client/server communications.
```

Comment:
I am assuming that @vasilvv will be able to update this sample code and that these will be the relevant URLs. This is something else that needs to be completed before this PR is merged.

Reply:
The client script should work with replacing the "quic-transport" scheme with "https".

Reply:
Gotcha. Is there an equivalent WebTransport over HTTP/3 server script that's available, though? Or is there a public server that Google or a standards group is running that folks could connect to?